source/retryable-reads/retryable-reads.rst

```diff
@@ -670,9 +674,11 @@ minor risk of degraded performance. Additionally, any customers experiencing
 degraded performance can simply disable ``retryableReads``.
 
 
-Changelog 
+Changelog
 ==========
 
+2022-01-07: Note that drivers should retry handshake network failures.
```

Comment:
This should have a corresponding last modified and version bump in the header. I assume a minor version bump is appropriate since this is a bit more than a typo/clarification.

Reply:
done!